content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1086,6 +1086,18 @@ has been truncated to 1000.
 
 If the number of backend endpoints falls below 1000, the control plane removes this annotation.
 
+### endpoints.kubernetes.io/last-change-trigger-time
+
+Type: Annotation
+
+Example: `endpoints.kubernetes.io/last-change-trigger-time: "2022-01-25T20:39:38Z"`
+
+Used on: Endpoints
+
+This annotation is set on an [Endpoints](/docs/concepts/services-networking/service/#endpoints) object
+that represents the timestamp (stored as RFC 3339 date-time string, e.g. '2018-10-22T19:32:52.1Z') of the last change.
+This timestamp belongs to either a Pod or Service object, that triggered the change in the endpoints object.
```

Comment:
```suggestion
This timestamp belongs to either a Pod or Service object, that triggered the change in the Endpoints object.
```
How does ”belong” fit into this sentence? I don't understand what it means.

Reply:
How does ”belong” fit into this sentence? I don't understand what it means.

Reply:
```
This annotation is set on an [Endpoints](/docs/concepts/services-networking/service/#endpoints) object
that represents the timestamp (stored as RFC 3339 date-time string, e.g. '2018-10-22T19:32:52.1Z') of the last change, of some Pod or Service object, that triggered the endpoints object change.
```
This one will be better

Reply:
> This timestamp belongs to either a Pod or Service object, that triggered the change in the Endpoints object.

I still don't quite follow.

Reply:
Agreed. This "belongs to" is confusing. Better revise.
/approve cancel